apartmenttherapy/oxygen-lib-ui
# Change post.post id to post.resourceID

#### What does this PR do & why?
- Changes references from `post.postID` to `post.resourceID`.
- See Oxy change here https://github.com/apartmenttherapy/oxygen/pull/1005

#### How can this be manually tested?
- Check SpotIM props
- Check PostBundle props
- Check PostActions props

#### Link to story card and other documentation
https://app.asana.com/0/937985426519793/936676642497469/f

#### GIF
![](https://media.giphy.com/media/l0MYLKgC47tnTGYW4/giphy.gif)

#### Have you remembered to...?
